src/pages/contributing/documentation.mdx

```diff
@@ -683,7 +733,7 @@ Identify the content elements within the component, for example, title, label, p
 
 Briefly explain any particular rules such as phrase structure, word limits, and character limits. Where there's more than one rule, use bullets to list out the multiple rules for ease of reading. If necessary, refer out to main content guidelines for more detailed information, for example, capitalization, recommended action labels.
 
-See [Checkbox](https://www.carbondesignsystem.com/components/checkbox/usage#content) and [Modal](https://www.carbondesignsystem.com/components/modal/usage#content) for examples.
+See the [Checkbox](/components/checkbox/usage#content) and [Modal](/components/modal/usage#content) components for examples.
```

Comment:
Question about links: if they copy or download the template will our shortened links still work outside the website?

Reply:
I had this same thought after I pushed this. Maybe it would be best moving forward to just have the full links displayed for all of our templates. It might be the safer approach.

Reply:
Okay I have updated all of the URLs to the normal ones. Moving forward I will update the other templates to have them too so it is more clear and also consistent.